engine/src/main/resources/assets/shaders/convertDepthToRGB_frag.glsl

```diff
@@ -0,0 +1,6 @@
+uniform sampler2D texSceneOpaqueDepth;
+
+void main() {
+    float depthOpaque = texture2D(texSceneOpaqueDepth, gl_TexCoord[0].xy).r;
+    gl_FragData[0] = vec4(depthOpaque,depthOpaque,depthOpaque,1.0);
```

Comment:
You need to linearize here. Depth data comes as 1/z. 

Try for example looking at this SO answer:
https://stackoverflow.com/questions/6652253/getting-the-true-z-value-from-the-depth-buffer